### DISCUSSION TITLE
no Shadows On Android Version.

### DISCUSSION THREAD

**Discussion by justlimeguy** [godotengine/godot-proposals#8210] [2D]
so, I tried the engine's shadow feature ( LightOccluder2D with point and directional light 2D) but as soon a i press shadows it just hides the light and does not show any thing. 

- the thing on windows 

https://github.com/godotengine/godot-proposals/assets/142280135/65c17068-9f97-4e0c-a555-1cc07f868cd4

- the thing on android 

https://github.com/godotengine/godot-proposals/assets/142280135/c320371d-11ba-4af0-ba4b-15a2ff8221fa

**Comment by AThousandShips** [comment]
This is for discussing features, not bug reporting, please report bugs over [here](https://github.com/godotengine/godot/issues)

**Reply by justlimeguy** [reply]
done. and sorry for the misunderstanding

**Reply by AThousandShips** [reply]
No worries!
